### PR TITLE
Add ParseProposalIDFromEvents helper function

### PR DESCRIPTION
**Pull request by ThanhNhann**
## Description

closes: #4809 

### Commit Message / Changelog Entry

```text
imp: add ParseProposalIDFromEvents helper function
```

see the [guidelines](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#commit-messages) for commit messages. (view raw markdown for examples)

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#pull-request-targeting)).
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/main/docs/build/building-modules/11-structure.md) and [Go style guide](../docs/dev/go-style-guide.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/ibc-go/blob/main/testing/README.md#ibc-testing-package).
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`).
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Provide a [commit message](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#commit-messages) to be used for the changelog entry in the PR description for review.
- [ ] Re-reviewed `Files changed` in the Github PR explorer.
- [ ] Review `Codecov Report` in the comment section below once CI passes.

### CODE REVIEW

File: testing/events.go [hunk 1]
```diff
@@ -135,6 +135,20 @@ func ParseAckFromEvents(events []abci.Event) ([]byte, error) {
 	return nil, fmt.Errorf("acknowledgement event attribute not found")
 }
 
+// ParseProposalIDFromEvents parses events emitted from MsgSubmitProposal and returns proposalID
+func ParseProposalIDFromEvents(events []abci.Event) (uint64, error) {
+	for _, event := range events {
+		for _, attribute := range event.Attributes {
+			if attribute.Key == "proposal_id" {
+				proposalID, err := strconv.ParseUint(attribute.Value, 10, 64)
```
Comment (damiannolan) [inline testing/events.go:143]: nit: I think you could return directly the call to `strconv.ParseUint` instead of using the vars

**Review by damiannolan — APPROVED** [review]
Thank you @ThanhNhann ❤️

File: testing/events.go (hunk 1 above)
Comment (ThanhNhann) [reply testing/events.go]: thank @damiannolan, updated!

**Review by crodriguezvega — APPROVED** [review]
Thank you, @ThanhNhann!

**Pull request merged**